testnet/stacks-node/src/config.rs

```diff
@@ -2406,7 +2406,7 @@ impl ConnectionOptionsFile {
                     .map_err(|e| format!("Invalid connection_option.public_ip_address: {}", e))
             })
             .transpose()?;
-        let mut read_only_call_limit = HELIUM_DEFAULT_CONNECTION_OPTIONS
+        let mut read_only_call_limit = LOCAL_TESTNET_DEFAULT_CONNECTION_OPTIONS
```

Comment:
I see here there are a lot move possible networks so in this context `LOCAL_TESTNET` would refer to more network besides `regtest` and `mocknet`. @ldiego08 I'm curious how you think this should be treated based on this. And thanks again for making this PR!

Reply:
@ASuciuX, while this function (and thus this constant) isn't used by runloops outside of regtest and mocknet (and some tests), other generic CLI commands like `stacks-node check-config` do utilize it. Therefore, it can be concluded that more networks are indeed involved here. Additionally, a function like `into_config` from a generic struct like `ConnectionOptionsFile` suggests a general-purpose use.

We could rename it to simply `DEFAULT_CONNECTION_OPTIONS` since there isn't another set of default connection options anyway. What do you think?

I would also suggest revisiting these config structs in a separate PR to refactor and document them for clarity.